docs/docs/references/ip_addresses.md

```diff
@@ -1,12 +0,0 @@
-# IP addresses to allow-list
```

Comment:
@jpetey75 we need to update sidebar.js as it has a reference to `references/ip_addresses`

Reply:
thanks @ZeRego done